start.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+chown -rf root:root ~/.ssh/config
+chmod -rf 644 ~/.ssh/config
```

Comment:
yes i am just troubleshooting before building it every time, it is my poor attempt to save time